jboss-kie-kieserver/tests/bats/jboss-kie-kieserver.bats

```diff
@@ -45,6 +45,26 @@ teardown() {
     [ "${RHPAM_XA_CONNECTION_PROPERTY_URL}" = "${RHPAM_URL}" ]
 }
 
+@test "test if the EJB_TIMER datasource has been auto-configured also with RHPAM_XA_CONNECTION_PROPERTY_Url" {
+    local expected_timer_service="EJB_TIMER"
+    local expected_datasources="EJB_TIMER,RHPAM"
+    local expected_url="jdbc:h2:/deployments/data/h2/rhpam;AUTO_SERVER=TRUE"
```

Comment:
The point of this issue is not exactly what you've changed, but instead, we need to make sure if the user sets the XA property for the main datasource, e.g. RHPAM_ XA_CONNECTION_PROPERTY_Url it is also set for EJBTIMER _XA_CONNECTION_PROPERTY_URL.

Reply:
on this bats tests focus on datasources that uses mariadb and postgresql driver.
Same for the other bats tests you've added.

Reply:
I added 2 tests specifically on mysql and postgres, are they enough?